docs/wiki/installation/install-osx.md

```diff
@@ -43,6 +43,16 @@ To update osquery:
 $ brew update
 $ brew upgrade osquery
 ```
+Post installation steps to run osqueryd if you have never installed and run osqueryd on this Mac.
+
+After completing the brew installation run the following commands. If you are using the chef recipe to install osquery then these steps are not necessary, the recipe has this covered.
```

Comment:
Maybe link to the chef recipe you're referencing, is this the chef we include in our docs? http://osquery.readthedocs.io/en/stable/deployment/configuration/#chef-os-x